components/src/core/index.ts

```diff
@@ -1,13 +1,11 @@
-import { ChannelValue } from './channel-value';
-import { EmptyState } from './empty-state';
-import { Header } from './header';
-import { Hero } from './hero';
-import { HeroBanner } from './hero-banner';
-import { InfoListItem } from './info-list-item';
-import { ScoreCard } from './score-card';
-import { wrapIcon } from './icon-wrapper/icon-wrapper';
+export { ChannelValue } from './channel-value';
+export { EmptyState } from './empty-state';
+export { Header } from './header';
+export { Hero } from './hero';
+export { HeroBanner } from './hero-banner';
+export { InfoListItem } from './info-list-item';
+export { ListItemTag, ListItemTagProps } from './list-item-tag';
```

Comment:
why are we exporting the Props type for this component?

Reply:
Just like why we are exporting component props in React Component Library...? I thought it might get handy in some TS projects. I am not sure why other components do not have the type exported

Reply:
I think it's only necessary to export the type if we expect someone to use it as a type, e.g.:
```
const MyProps:ListItemTagProps = {
    label: 'Hello',
}
...
<ListItemTag {...MyProps}/>
```

If you just want intellisense to be able to tell you what the available props are on a component, we get that from just the component export.

I guess it doesn't hurt anything to export the types too, I was just curious if there was a particular reason you were exporting this one when the other components don't. If you want to export all of the other ones too, that's fine.

Reply:
All props exposed